ui/ts/pages/log.ts

```diff
@@ -115,7 +132,7 @@ module AdminViews {
 
       function onChangeSeverity(val: string): void {
         entries.level(val);
-        entries.refresh();
+        m.route(entries.buildURL());
```

Comment:
This seems like the best way to handle this for now; the router is simple and straightforward, but not especially capable when it comes to updating state.